File: hooks/tests/fixtures/codenav/julia/navigation.jl
module Reads
using Statistics
import LinearAlgebra: norm
include("helpers.jl")
abstract type AbstractRead end
struct Read <: AbstractRead
    sequence::String
end
function normalize(x::Vector{Float64})
    norm(x) + helper(x)
end
score(x) = normalize(x)
end

File: hooks/tests/fixtures/codenav/fortran/navigation.f90
MODULE Reads
  USE iso_fortran_env
  INCLUDE 'constants.inc'
  implicit none
  type :: BaseRead
    real :: quality
  end type BaseRead
  type, extends(BaseRead) :: Read
    integer :: length
  end type Read
contains
  SUBROUTINE Normalize(x)
    real, intent(inout) :: x
    CALL Helper(x)
    x = Score(x)
  end subroutine Normalize
  real FUNCTION Score(x) result(y)
    real, intent(in) :: x
    y = abs(x)
  end function Score
end module Reads
! subroutine phantom()
